web/src/settings_users.js

```diff
@@ -150,23 +150,47 @@ function role_selected_handler(event, dropdown, widget) {
     widget.render();
 }
 
-function get_roles() {
+function get_roles(user_ids) {
+    return get_roles_with_counts(user_ids);
```

Comment:
Noticing that `get_roles` and `get_roles_with_counts` do similar things. Should we consider removing `get_roles` and just stick with `get_roles_with_counts`?

Reply:
Welcome! Just a suggestion: aim to show the maintainers the best work possible to save them time.